### PR TITLE
Update HCC docs

### PR DESCRIPTION
**Pull request by isc-pbarton**
Updates to Health Connect Cloud documentation.

### CODE REVIEW

File: docs/hcc.md [hunk 1]
```diff
@@ -1,42 +1,105 @@
...
-
-A Health Connect Cloud Dev deployment will have a main namespace (which is also present in Test and Prod) and should also have a dedicated namespace for each developer contributing. These namespaces can be configured via the Cloud Portal. It is important for changes to originate in developer-specific namespaces; the main namespace in the dev deployment should always be kept up to date with the development branch in GitLab, either through a CI/CD pipeline or by use of the Git Pull page from git-source-control.
+# Change Control with Embedded Git for Health Connect Cloud
+
+## General 
+The recommended workflow for Health Connect Cloud is to: create an interface branch, make changes, sync them to GitLab, create a merge request, review and approve the merge request, and then pull the changes into a protected namespace.
```
Comment (isc-tleavitt) [inline docs/hcc.md:4]: Nit: "pull the changes" sounds like an action but is automatic with CI/CD.

File: docs/hcc.md [hunk 2]
```diff
@@ -1,42 +1,105 @@
...
+![Step 4: Merge request and repeat 2-3](images/hcc/hcc-step-4.png)
+
+A Health Connect Cloud Dev deployment will have at least one Protected Namespace and should also have a dedicated User Namespace for each user. These namespaces can be configured via the Cloud Portal for the selected Health Connect Cloud deployment. It is important for changes to originate in User Namespaces. Protected Namespaces in the Dev deployment are automatically kept up to date with the development branch of the associated GitLab repo through a CI/CD pipeline.
+
+# Prerequisites: InterSystems Responsibilities
+ - Create the GitLab repository for the organization
+ - Grant appropriate access and privileges to the GitLab repository
+ - Onboard customers on source control tools for Health Connect Cloud
+ - Maintain change control documentation and videos
+   - [Official Change Control Documentation](https://docs.intersystems.com)
```
Comment (isc-tleavitt) [inline docs/hcc.md:24]: Can we somehow comment this out for now? (Until we have more official documentation to point to more specifically?)

**Review by isc-tleavitt — CHANGES_REQUESTED** [review]
a few really minor points here. I've also passed along to DP PM to review but that review shouldn't hold it up, we can go with a followup if needed

**Comment by isc-pbarton** [comment]
I've made those two changes and put it back for your review

**Review by isc-tleavitt — APPROVED** [review]

**Comment by isc-tleavitt** [comment]
Thank you @isc-pbarton! We'll probably get some more feedback from DP PM but merging as this is a great step forward.

**Pull request merged**